src/geometry_aux.cpp

```diff
@@ -40,6 +42,71 @@ void update_universe_cell_count(int32_t a, int32_t b)
   }
 }
 
+double calculate_optical_thickness(Position start_sampled_position,
+  Position end_sampled_position, uint64_t* seed)
```

Comment:
you don't need a `seed` here
```suggestion
  Position end_sampled_position)
```